Gems/Atom/Tools/MaterialCanvas/Code/Source/Document/MaterialCanvasDocument.cpp

```diff
@@ -143,21 +143,18 @@ namespace MaterialCanvas
         // filters is used to determine how the create document dialog is populated. The base document class rejects file types that are not
         // listed in the extension supported for opening. Will change to make the base class support opening anything listed in open
         // or create and the create dialog look at the create list exclusively.
-        documentType.m_supportedExtensionsToCreate.push_back({ "Material Canvas Template", "materialcanvastemplate.azasset" });
-        documentType.m_supportedExtensionsToCreate.push_back({ "Material Canvas", "materialcanvas.azasset" });
-        documentType.m_supportedExtensionsToOpen.push_back({ "Material Canvas Template", "materialcanvastemplate.azasset" });
-        documentType.m_supportedExtensionsToOpen.push_back({ "Material Canvas", "materialcanvas.azasset" });
-        documentType.m_supportedExtensionsToSave.push_back({ "Material Canvas", "materialcanvas.azasset" });
-
-        // Currently using AnyAsset As a placeholder until proper asset types are created.
-        documentType.m_supportedAssetTypesToCreate.insert(azrtti_typeid<AZ::RPI::AnyAsset>());
+        documentType.m_supportedExtensionsToCreate.push_back({ "Material Graph Template", "materialgraphtemplate" });
+        documentType.m_supportedExtensionsToCreate.push_back({ "Material Graph", "materialgraph" });
+        documentType.m_supportedExtensionsToOpen.push_back({ "Material Graph", "materialgraph" });
+        documentType.m_supportedExtensionsToSave.push_back({ "Material Graph", "materialgraph" });
```

Comment:
nit: These hardcoded strings are repeated a lot, lets make named constants

Reply:
I will replace these with variables fit into the build function after I split the document class into a generic graph document class handy material graph compiler. I should be doing that soon which will also support the pass editor.